File: NBA_Stats.py
import requests
import discord
import json

with open("config.json", "r") as f:
    config = json.load(f)

NBA_url = config['url']
TOKEN = config['chatbot_token']
headers = config['headers']

querystring = {"page":"0","per_page":"25","search":"Michael jordan"}

response = requests.request("GET", NBA_url, headers=headers, params=querystring)

print(response)

class MyClient(discord.Client):
    async def on_ready(self):
        print('Logged in as')
        print(self.user.name)
        print(self.user.id)
        print('------')

intents = discord.Intents.default()
intents.message_content = True

client = MyClient(intents=intents)
client.run(TOKEN)

File: NBA_Stats.js
const {Client,Events, GatewayIntentBits} = require('discord.js');
const config = require('./config.json');
const TOKEN = config.chatbot_token;
const {Team_Stats} = require('./Team_stats.js');

// //To bring result from Stats.py
// const spawn = require("child_process").spawn;
// const pythonProcess = spawn('python',['./Stats.py']);

const client = new Client({ intents: [GatewayIntentBits.Guilds, GatewayIntentBits.GuildMessages, GatewayIntentBits.MessageContent] });

client.on("message", () =>{
    console.log("The AI bot is online"); //message when bot is online
    console.log(`Logged in as ${client.user.tag}!`);
});


client.on("messageCreate", (message) => {

    let commands = message.content.split(" ");

    if (commands[0].toLowerCase() === "!team") {
        //Output information of team id that is input after !Team
        message.channel.send(`Information of your following team`);

        //Send city name to Team_stats.py
        let cityName = JSON.stringify(commands[1]);
        console.log(cityName);

        const data = Team_Stats
        console.log(data);
    }

    else if (message.content.substring(0,8).toLowerCase() == "!Player") {
        message.channel.send(`The following player`);
    }
});


client.login(TOKEN);